pkg/otlp/model/translator/exponential_histograms_translator.go

```diff
@@ -147,6 +148,14 @@ func (t *Translator) mapExponentialHistogramMetrics(
 			agentSketch.Basic.Sum = histInfo.sum
 			agentSketch.Basic.Avg = agentSketch.Basic.Sum / float64(agentSketch.Basic.Cnt)
 		}
+		if delta && p.HasMin() {
+			// override min if available and delta.
```

Comment:
I'd recommend either:
a) removing the comment because it's pretty obvious what's going on
b) rephrasing it to `// override min if it is available and aggregation temporality is delta`.

Reply:
Ok, I just removed them, agree that it's pretty obvious